pkg/s3utils/utils.go

```diff
@@ -135,6 +135,14 @@ func GetRegionFromURL(endpointURL url.URL) string {
 	return ""
 }
 
+// isAliyunOSSEndpoint - Match if it is exactly Aliyun OSS endpoint.
+func isAliyunOSSEndpoint(endpointURL url.URL) bool {
```

Comment:
Export this as a public function @hanqingwu ?

Reply:
no,  only be used in  IsVirtualHostSupported now.